### DISCUSSION TITLE
Guildes

### DISCUSSION THREAD

**Discussion by OrionYp** [Crownicles/Crownicles#2522] [Nouvelles idées]
Passé le niveau 100 il n'y a plus vraiment d'utilité à améliorer sa guilde, je propose donc d'ajouter 2 trucs : 

--- La possibilité d'amélioration l'entrepôt "+5 emplacement de bonbons pour 5 000💰" "+ 1 empl de salade/viande pour 10 000" "+ 1 emplacement de soupe pour 20 000"
Avec un max de : 
- pour guildes lvl <100 → 25🍬, 15🥬🥩, 5🍲

- pour guildes lvl 100 à 110 → 30🍬, 20🥬🥩, 10🍲

- pour guildes lvl 110 à 120 → 35🍬, 25🥬🥩, 10🍲

- pour guildes lvl 120 à 130 → 40🍬, 30🥬🥩, 15🍲
etc.

--- La possibilité d'améliorer l'abri : 
Au prix de 50 000 par emplacement,
- pour guildes lvl <100 → 6 emplacements

- pour guildes lvl 100 à 110 → 7 emplacements

- pour guildes lvl 110 à 120 → 8 emplacements 

- pour guildes lvl 120 à 130 → 9 emplacements
etc.

---
Si cela vous semble cher pensez que rien n'oblige à les acheter, si on n'achète rien on a les mêmes places qu'actuellement, et les ajouts sont pour les guildes de lvl 100 ou + qui ont généralement des membres riches qui ne savent plus quoi faire de leur 💰.

**Comment by Carsebel** [comment]
Pourquoi payer?
Tous les joueurs ne sont pas forcément riches dans les guildes lvl+100

**Reply by OrionYp** [reply]
Pour avoir des objectifs en plus pour ceux qui ne visent pas les top

**Reply by Cirpacha** [reply]
Comme on paye déjà l'xp pour up la guilde, il vaudrait mieux soit pouvoir débloquer les emplacements dès le début en payant, soit les avoir automatiquement passé un certain level de guilde (sans payer donc)

**Comment by Procion06** [comment]
je propose que ce soit un bonus (et une carotte) pour motiver et récompenser les joueurs qui up leur guilde :

Guilde niveau 100 : +3 places dans l'abri > 9 emplacements au total
Guilde niveau 150 : +3 places dans l'abri > 12 emplacements au total

pas de thune à sortir, juste une récompense concrète et méritée pour les efforts fournis par les joueurs, qui change du badge en carton
n'oubliez pas que ça prend une éternité avant d'arriver au niveau 150 et que c'est loin d'être déconnant 

Votez en masse si cette idée vous plaît !

**Reply by Ntalcme** [reply]
L'idée est bien, vraiment bien surtout quand on sait les prochains ajouts mais la "récompense" de up les guildes c'est d'avoir des meilleurs guild daily.
Peut-être faudrait juste revoir les récompenses de GD qui sont assez médiocres quand l'on compare la quantité de temps et d'investissement requis, surtout après le level 100.

**Reply by Ntalcme** [reply]
Mais sinon je suis pour, 6 c'est trop peu.

**Comment by Ravenclaw-l** [comment]
je me permets de revenir sur le sujet de l'agrandissement du shelter. 
Maintenant que les familiers sont utiles en combat, l'abri est vraiment trop étroit.

**Reply by Ntalcme** [reply]
En vrai si on considère que chaque membre a une place dans le shelter et sur lui ça fait 2 places, c'est là justement qu'il va falloir faire des choix sur quel pet garder ou non, une vraie stratégie.
**Toutefois**, on devrait quand même augmenter un peu le shelter sans trop l'augmenter et sur ça je rejoins un peu l'idée de Procion plus haut.

Une guilde devrait avoir 12 places au maximum (2/membres on va dire) mais au lieu de faire +3 à 100 et +3 à 150 je propose ceci : 
- 100 : +1 
- Chaque 10 levels : +1
